### PR TITLE
chore: add ios e2e test in CI

### PR DESCRIPTION
**Pull request by GuillaumeEgret**
PR to run ios test in github action and add a report

### CODE REVIEW

File: packages/ios-poc/test.yaml [hunk 1]
```diff
@@ -1,3 +1,4 @@
 appId: org.reactjs.native.example.fakeStore
 ---
 - tapOn: "VIENS ON PETE TOUT"
+- assertVisible: "ON A TOUT PETE"
```
Comment (Almouro) [inline packages/ios-poc/test.yaml]: :D

File: packages/ios-poc/src/launchIOS.ts [hunk 2]
```diff
@@ -84,7 +84,9 @@ const launchTest = async ({
     stdio: "inherit",
   });
   const recordingPromise = startRecord(simulatorId, traceFile);
-
+  execSync(`sleep 2`, {
```
Comment (Almouro) [inline packages/ios-poc/src/launchIOS.ts:87]: could you add a command on why we need to sleep here?

File: packages/ios-poc/src/launchIOS.ts [hunk 3]
```diff
@@ -47,7 +47,7 @@ const executeAsyncCommand = (command: string): Promise<void> => {
 };
 
 const startRecord = (simulatorId: string, traceFile: string): Promise<void> => {
-  const templateFilePath = `${__dirname}/../Flashlight.tracetemplate`;
+  const templateFilePath = `${__dirname}/../tp.tracetemplate`;
```
Comment (Almouro) [inline packages/ios-poc/src/launchIOS.ts:50]: should we delete `Flashlight.tracetemplate` then? I liked having Flashlight in the name to be honest :D

File: .github/workflows/ios_e2e.yml [hunk 4]
```diff
@@ -37,3 +37,4 @@ jobs:
             ## Flashlight iOS Report 🔦
 
             - [🚀 Report](${{ env.NETLIFY_PREVIEW_URL }})
+          append: true
```
Comment (Almouro) [inline .github/workflows/ios_e2e.yml:40]: why the append? I think by default it edits previous message

File: .github/workflows/fakeStore.app/_CodeSignature/CodeResources [hunk 5]
```diff
@@ -131,7 +131,7 @@
 		</data>
 		<key>main.jsbundle</key>
 		<data>
-		lJdjPsLiXuiKCW802TFgvJ9wa30=
+		ozrUYoBQ5OeS6gGjvWZiq0+ZR7g=
```
Comment (Almouro) [inline .github/workflows/fakeStore.app/_CodeSignature/CodeResources:134]: should we go back to 40?

Considering the report feels a bit empty, wdyt? 

image

File: packages/ios-poc/src/launchIOS.ts (hunk 3 above)
Comment (GuillaumeEgret) [reply packages/ios-poc/src/launchIOS.ts:50]: I will rename the template to have Flashlight again

File: .github/workflows/ios_e2e.yml (hunk 4 above)
Comment (GuillaumeEgret) [reply .github/workflows/ios_e2e.yml:40]: I thought it would be better because now we compare with the previous run. I can remove it

Comment (GuillaumeEgret) [reply .github/workflows/ios_e2e.yml]: Removed

File: .github/workflows/fakeStore.app/_CodeSignature/CodeResources (hunk 5 above)
Comment (GuillaumeEgret) [reply .github/workflows/fakeStore.app/_CodeSignature/CodeResources]: Changed to 35

File: packages/ios-poc/src/launchIOS.ts (hunk 2 above)
Comment (GuillaumeEgret) [reply packages/ios-poc/src/launchIOS.ts]: Refacto

File: packages/ios-poc/src/launchIOS.ts (hunk 3 above)
Comment (GuillaumeEgret) [reply packages/ios-poc/src/launchIOS.ts]: Done

**Review by Almouro — APPROVED** [review]

**Pull request merged**